### PR TITLE
[otbn] Tidy up long lines and unnecessary imports in Python code

### PR DESCRIPTION
**Pull request by rswarbrick**
Signed-off-by: Rupert Swarbrick <rswarbrick@lowrisc.org>

### CODE REVIEW

File: hw/ip/otbn/util/check_loop.py [hunk 1]
```diff
@@ -194,6 +197,7 @@ def _check_loop_stack(program: OTBNProgram,
 
     The second condition in the list, distinct end addresses, is checked
     separately.
+
```
Comment (jadephilipoom) [inline hw/ip/otbn/util/check_loop.py:200]: nit: extra newline?

**Review by jadephilipoom — APPROVED** [review]
LGTM!

File: hw/ip/otbn/util/check_loop.py (hunk 1 above)
Comment (rswarbrick) [reply hw/ip/otbn/util/check_loop.py:200]: Ah, that's just the default formatting from the Emacs Python mode. Looks like it doesn't match the recommendations from PEP8 or PEP-0257 (I just looked!) so I'll drop it and try to figure out how to configure things properly :-)

Comment (rswarbrick) [reply hw/ip/otbn/util/check_loop.py:200]: For any other Emacs users (or future me!), the thing to configure is the `python-fill-docstring-style` variable. It defaults to `'pep-257` and to lose the trailing newline it should be set to `'pep-257-nn`.

For any other Emacs users who happen to use Projectile (who am I kidding? This is clearly just me!), something like this works nicely:
```emacs-lisp
(defun rjs/python-setup ()
  ;; Use 79-column wrapping in Python files (PEP8)
  (setq fill-column 79)

  (when (string= (projectile-project-name) "opentitan")
    ;; pep-257-nn mode is almost the same as the default, but it drops
    ;; the trailing newline in a multi-line docstring (matching the
    ;; official Python style guides). It's what we officially adopt in
    ;; OpenTitan, even if it's not as pretty!
    (setq-local python-fill-docstring-style 'pep-257-nn)))

(with-eval-after-load "python" (add-hook 'python-mode-hook 'rjs/python-setup))
```

**Comment by rswarbrick** [comment]
@GregAC: Would you mind taking a look at this? It should be reasonably trivial, but clashes with OTBN DV fixes so it would be nice to get it in.

**Review by GregAC — APPROVED** [review]

**Pull request merged**